checkov/terraform/checks/graph_checks/azure/AzureContainerInstanceconfigManagedIdentity.yaml

```diff
@@ -0,0 +1,17 @@
+metadata: 
+  id: "CKV2_AZURE_28"
+  name: "Ensure Container Instance is configured with managed identity"
+  category: "GENERAL_SECURITY"
+
+definition:
+  and:
+    - cond_type: attribute
+      resource_types: azurerm_container_group
+      attribute: identity.type
+      operator: "exists"
```

Comment:
same here, not needed, but please test it 😄

Reply:
Hello @gruebel,

I tested it and this block is mandatory in this policy. It doesn't work without it.